lms/static/coffee/spec/instructor_dashboard/send_email_spec.coffee

```diff
@@ -0,0 +1,82 @@
+                send_to: JSON.stringify([@send_email.$send_to.first().val()]),
+                subject: testSubject,
+                message: testBody,
+            },
+            success: jasmine.any(Function),
+            error: jasmine.any(Function),
+        }
+
+    it 'binds the send email button to a click event', ->
+        expect(@send_email.$btn_send).toHandle 'click'
+
+    it 'cannot send an email with no target', ->
+        spyOn(window, "alert")
+        spyOn($, "ajax")
+        for target in @send_email.$send_to
+            target.checked = false
+        @send_email.$btn_send.click()
+        expect(window.alert).toHaveBeenCalledWith("Your message must have at least one target.")
+        expect($.ajax).not.toHaveBeenCalled()
+
+    it 'cannot send an email with no subject', ->
+        spyOn(window, "alert")
+        spyOn($, "ajax")
+        @send_email.$subject.val("")
+        @send_email.$btn_send.click()
+        expect(window.alert).toHaveBeenCalledWith("Your message must have a subject.")
+        expect($.ajax).not.toHaveBeenCalled()
+
+    it 'cannot send an email with no message', ->
+        spyOn(window, "alert")
+        spyOn($, "ajax")
+        @send_email.$emailEditor =
+            save: ->
+                {"data": ""}
+        @send_email.$btn_send.click()
+        expect(window.alert).toHaveBeenCalledWith("Your message cannot be blank.")
+        expect($.ajax).not.toHaveBeenCalled()
+
+    it 'can send a simple message to a single target', ->
+        spyOn($, "ajax").and.callFake((params) =>
+          params.success()
+        )
+        @send_email.$btn_send.click()
+        expect($('.msg-confirm').text()).toEqual('Your email was successfully queued for sending. Please note that for large classes, it may take up to an hour (or more, if other courses are simultaneously sending email) to send all emails.')
+        expect($.ajax).toHaveBeenCalledWith(@ajax_params)
+
+    it 'can send a simple message to a multiple targets', ->
+        spyOn($, "ajax").and.callFake((params) =>
+            params.success()
+        )
+        @ajax_params.data.send_to = JSON.stringify(target.value for target in @send_email.$send_to)
+        for target in @send_email.$send_to
+            target.checked = true
+        @send_email.$btn_send.click()
+        expect($('.msg-confirm').text()).toEqual('Your email was successfully queued for sending. Please note that for large classes, it may take up to an hour (or more, if other courses are simultaneously sending email) to send all emails.')
+        expect($.ajax).toHaveBeenCalledWith(@ajax_params)
+
+    it 'can handle an error result from the bulk email api', ->
+        spyOn($, "ajax").and.callFake((params) =>
+            params.error()
+        )
+        spyOn(console, "warn")
+        @send_email.$btn_send.click()
+        expect($('.request-response-error').text()).toEqual('Error sending email.')
+        expect(console.warn).toHaveBeenCalled()
```

Comment:
Is there a reason this new spec needs to be Coffeescript? We'd like to not create any new coffeescript files if avoidable - [see here](https://openedx.atlassian.net/wiki/display/FEDX/How+to+add+a+new+page+to+LMS+or+Studio#HowtoaddanewpagetoLMSorStudio-UseJavaScriptinsteadofCoffeeScript)